app/models/request.rb

```diff
@@ -0,0 +1,5 @@
+class Request < ActiveRecord::Base
+  validates :content presence: true
+  validates :requester_id, :responder_id, presence: true, numericality: { only_integer: true }
```

Comment:
Hi. Two things:

1. It is better for us to validate a `requester` instead of a `requester_id`. This ensures that the requester exists. 

2. We can't require that the `responder` is present when we create a request, since there won't be anyone assigned.